---a/net/minecraft/enchantment/EnchantmentArrowKnockback.java
+++b/net/minecraft/enchantment/EnchantmentArrowKnockback.java
@@ -2,29 +2,25 @@
 
 import net.minecraft.util.ResourceLocation;
 
-public class EnchantmentArrowKnockback extends Enchantment
-{
+public class EnchantmentArrowKnockback extends Enchantment {
+
     private static final String __OBFID = "CL_00000101";
 
-    public EnchantmentArrowKnockback(int enchID, ResourceLocation enchName, int enchWeight)
-    {
+    public EnchantmentArrowKnockback(int enchID, ResourceLocation enchName, int enchWeight) {
         super(enchID, enchName, enchWeight, EnumEnchantmentType.BOW);
         this.setName("arrowKnockback");
     }
 
-    public int getMinEnchantability(int enchantmentLevel)
-    {
+    public int getMinEnchantability(int enchantmentLevel) {
         return 12 + (enchantmentLevel - 1) * 20;
     }
 
-    public int getMaxEnchantability(int enchantmentLevel)
-    {
+    public int getMaxEnchantability(int enchantmentLevel) {
         return this.getMinEnchantability(enchantmentLevel) + 25;
     }
 
-    public int getMaxLevel()
-    {
+    public int getMaxLevel() {
         return 2;
     }
-}
+}
